contrib/format-hdf5/src/main/java/org/apache/drill/exec/store/hdf5/HDF5BatchReader.java

```diff
@@ -1027,16 +1074,17 @@ private void getAndMapCompoundData(String path, List<String> fieldNames, IHDF5Re
         }
       } else {
         int index = 0;
+        List<String> compoundDataTypes = new ArrayList<>();
```

Comment:
Could you please point out where this list is used in code?

Reply:
Removed this variable.